GenerateCDT/GenerateCDT.py

```diff
@@ -40,8 +144,8 @@ def main():
 	data.append(0x12)			# Block ID
 	data.append(0x22)			# Pulse length (L)
 	data.append(0x0B)			# Pulse length (H)
-	data.append(0xFF)			# Pulse count (L)
-	data.append(0x03)			# Pulse count (H)
+	data.append(0x00)			# Pulse count (L)
+	data.append(0x04)			# Pulse count (H)
```

Comment:
I incremented the pulse count here because the emulator I'm using (RetroVirtualMachine) doesn't implement tones with an odd number of pulses correctly for some reason. This change doesn't seem to affect loading on real hardware.